OverlayPlugin.Core/MemoryProcessors/Aggro/AggroMemory60.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace RainbowMage.OverlayPlugin.MemoryProcessors.Aggro
+{
+    class AggroMemory60 : AggroMemory
+    {
+        private const int aggroEnmityOffset = -2336;
+
+        // Aggro uses the same signature as Enmity

```

Comment:
Not sure why this conversation was resolved, weird.

The reasoning behind not just using EnmityMemory here is because the only way to do so would be to pull the IntPtr address, which doesn't really feel right.